### DISCUSSION TITLE
webpack Empty Context

### DISCUSSION THREAD

**Discussion by Cabezagithub** [quasarframework/quasar#16338] [General - Components / Directives / etc]
I'm trying to load JSON setup from folders. Each folder represent a Company and each Company have yours JSON setup with parameters.
The Company number came into URL, example: https://subdomain.domain.com?company_id=19,  other:   https://subdomain.domain.com?company_id=7

I have a folder with JSON setups:
/somefolderapp/company/NN/index.js (JSON with configuration)    
Where NN are the company ID, exemple:  /somefolderapp/company/19/index.js 
   index.js ->
module.exports = {
  logo: '/img/retras.png',
  sec: '/thetypeing/werr.pdf'
}

In the setup I have got: 
set () {
  const route = useRouter()
  const companyId = ref(route.query.company_id)     //  the parameters came from   URL  (route.query   parameters)
  const txt = `../../somefolder/${companyId}/index.js`     // Where   /somefolder  have inside all JSON with company setup
  const companysetup = require(txt)     // here have an error  "webpack Empty Context"
  ...
}

Basically I'm trying to have a one single application where I show and setup the screens according to parameters that come in the URL identifying the companies and reading their own JSON configuration files.  

Error: I can't to use  "require(variable)"  with a variable.
Note: I can use require('folder_some_fixed_txt')

Reading something, I tried to execute a "require(variable)"  on  "mounted()" method  but it doesn't work either.

I appreciate any help regarding this. Thank you!
